src/main/java/org/isf/video/gui/PatientPhotoPanel.java

```diff
@@ -56,6 +56,8 @@
 
 public class PatientPhotoPanel extends JPanel {
 
+	private static final String JPEG_FORMAT = "jpeg";
```

Comment:
My memory tells me that `jpg` is the most common usage in the project and not `jpeg`.

Reply:
actually at line 147 they are used all of them... maybe not need to have a constant?
`String[] extensions = { "tif", "tiff", JPEG_FORMAT, "jpeg", "bmp", "png", "gif" };`